homestyle.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+
+* {
+	box-sizing: border-box;
+}
+
+body { 
+	margin: 0; 
+	padding: 0;
+	font-family: 'Montserrat', sans-serif;
+	background-color: black;
+}
+
+/* Navbar */
+
+.navbar {
+	width: auto;
+	max-width: 100%;
+	max-height: auto;
+	color: white;
+	background-color: black;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 10px;
+	text-decoration: none;
+	letter-spacing: 3px;
+	font-size: 25px;
+}
+
+#NC {
```

Comment:
Do you need to have the background-color set as black since the nav bar container is already black?